packages/app/src/store/onboarding/types.ts

```diff
@@ -32,11 +45,17 @@ export interface OnboardingState {
   appURL?: URL;
   magicRecoveryCode?: string;
   username?: string;
+  onboardingInProgress?: boolean;
```

Comment:
Shouldn't this be something like `string`, or even better, one of `ScreenPaths`? That way, we can bump them back to the exact page they left off on?

Reply:
This is more of a check so that the redirect that is implemented [here](https://github.com/blockstack/ux/pull/288/files#diff-f0526fdc974eb1f648bac3b9dd5f5a78R49) doesn't activate the moment a key is generated